package.json

```diff
@@ -85,6 +86,7 @@
     "test:dist": "cross-env NODE_ENV=production yarn test",
     "test-integration": "jest tests_integration",
     "lint": "cross-env EFF_NO_LINK_RULES=true eslint . --format node_modules/eslint-friendly-formatter",
+    "lint-docs": "prettylint {.,docs,website}/*.md",
```

Comment:
Would be really good if we could run --fix and auto-commit changes. Because if people contribute via GitHub UI we don't want the build to fail.

Reply:
The auto-commit is available but it's dangerous since we'll need to expose our token to PR branches in the Travis, an alternative would be to enable the `--no-prose-wrap` option to reduce the possibility of linting errors.